spec/directives/use/error/load.hrx

```diff
@@ -0,0 +1,314 @@
+<===> unknown_scheme/input.scss
+@use "scheme:bar";
+
+<===> unknown_scheme/error
+Error: Can't find stylesheet to import.
+  ,
+1 | @use "scheme:bar";
+  | ^^^^^^^^^^^^^^^^^
+  '
+  /sass/spec/directives/use/error/load/unknown_scheme/input.scss 1:1  root stylesheet
+
+<===>
+================================================================================
+<===> no_extension/input.scss
+@use "other";
+
+<===> no_extension/other
+a {b: c}
+
+<===> no_extension/error
+Error: Can't find stylesheet to import.
+  ,
+1 | @use "other";
+  | ^^^^^^^^^^^^
+  '
+  /sass/spec/directives/use/error/load/no_extension/input.scss 1:1  root stylesheet
+
+<===>
+================================================================================
+<===> missing/input.scss
+@use "other";
+
+<===> missing/error
+Error: Can't find stylesheet to import.
+  ,
+1 | @use "other";
+  | ^^^^^^^^^^^^
+  '
+  /sass/spec/directives/use/error/load/missing/input.scss 1:1  root stylesheet
+
+<===>
+================================================================================
+<===> conflict/partial/input.scss
+// This import can't be resolved because it could refer to either the partial or
+// the non-partial file.
+@use "other";
+
+<===> conflict/partial/_other.scss
+a {partial: true}
+
+<===> conflict/partial/other.scss
+a {partial: false}
+
+<===> conflict/partial/error
+Error: It's not clear which file to import. Found:
+  /sass/spec/directives/use/error/load/conflict/partial/_other.scss
+  /sass/spec/directives/use/error/load/conflict/partial/other.scss
+  ,
+3 | @use "other";
+  | ^^^^^^^^^^^^
+  '
+  /sass/spec/directives/use/error/load/conflict/partial/input.scss 3:1  root stylesheet
+
+<===>
+================================================================================
+<===> conflict/extension/sass_and_scss/input.scss
+// This import can't be resolved because it could refer to either the ".sass" or
+// ".scss" file.
+@use "other";
+
+<===> conflict/extension/sass_and_scss/other.sass
+a
+  syntax: sass
+
+<===> conflict/extension/sass_and_scss/other.scss
+a {syntax: scss}
+
+<===> conflict/extension/sass_and_scss/error
+Error: It's not clear which file to import. Found:
+  /sass/spec/directives/use/error/load/conflict/extension/sass_and_scss/other.sass
+  /sass/spec/directives/use/error/load/conflict/extension/sass_and_scss/other.scss
+  ,
+3 | @use "other";
+  | ^^^^^^^^^^^^
+  '
+  /sass/spec/directives/use/error/load/conflict/extension/sass_and_scss/input.scss 3:1  root stylesheet
+
+<===>
+================================================================================
+<===> conflict/extension/scss_and_css/input.scss
+// This import can't be resolved because it could refer to either the ".sass" or
+// ".css" file.
```

Comment:
Same question here as I added to https://github.com/sass/dart-sass/pull/603. Might cause confusion on your second build?